src/LiveComponent/templates/form_theme.html.twig

```diff
@@ -6,8 +6,14 @@
 {%- endblock live_collection_widget -%}
 
 {%- block live_collection_entry_row -%}
-    {{ block('form_row') }}
+    {%- if button_move_up is defined and not button_move_up.rendered -%}
+        {{ form_row(button_move_up) }}
+    {%- endif -%}
+    {%- if button_move_down is defined and not button_move_down.rendered -%}
+        {{ form_row(button_move_down) }}
+    {%- endif -%}
     {%- if button_delete is defined and not button_delete.rendered -%}
         {{ form_row(button_delete) }}
     {%- endif -%}
+    {{ block('form_row') }}
```

Comment:
This would be a BC break as it would set the delete button before, when is was after for people already using this theme.